core/test/core-api/theming.spec.js

```diff
@@ -0,0 +1,65 @@
+const chai = require('chai');
+const assert = chai.assert;
+const expect = chai.expect;
+const sinon = require('sinon');
+
+import { LuigiConfig, LuigiTheming } from '../../src/core-api';
+
+describe('Core API - Theming', function() {
+  this.retries(1);
+  const getMockTheming = () => {
+    return {
+      themes: [
+        { id: 1, name: 'light' },
+        { id: 2, name: 'dark' },
+        { id: 3, name: 'blue' },
+        { id: 4, name: 'green' },
+        { id: 5, name: 'red' },
+        { id: 6, name: 'orange' },
+        { id: 7, name: 'yellow' }
```

Comment:
If colors are just labels and nothing more, maybe we should already provide the meaning of those labels? e.g. "yellow" should apply yellow colors to the core part of the Luigi? Orange and red almost the same, we could remove it. Or let's catch up on this once you are back @maxmarkus

Reply:
Its a spec file, so its a test and not an example. Guess its ok for that.